I've finished this task. Can you please check it?

IvanIKS opened the pull request.

Rommelua commented on src/main/java/core/basesyntax/Application.java:
```diff
@@ -3,5 +3,13 @@
+            ballArray[i] = lottery.getRandomBall();
+        }
+        for (Ball ball : ballArray) {
+            System.out.println(ball.toString());
```
```suggestion
            System.out.println(ball);
```

Rommelua commented on src/main/java/core/basesyntax/Color.java:
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
+    Black,
+    White,
+    Red,
+    Blue,
+    Green,
+    Yellow,
```
uppercase

Rommelua commented on src/main/java/core/basesyntax/ColorSupplier.java:
```diff
@@ -1,7 +1,12 @@
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/6b971c2f-6117-4f5a-8758-aa3e4d9f1468)

Rommelua commented on src/main/java/core/basesyntax/ColorSupplier.java:
```diff
@@ -1,7 +1,12 @@
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return "" + color;
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/6f7776b7-5ffb-4540-9d51-75fd488a3d2f)

Rommelua reviewed (changes requested):
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there.

IvanIKS commented:
I have revorked the code in the way you suggested: used "ball" instead of "ball.toString", changed text to uppercase, used .name() and moved Random to class level. Can you please check the task again?

Rommelua commented on src/main/java/core/basesyntax/Lottery.java:
```diff
@@ -0,0 +1,14 @@
+        Random random = new Random();
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(100));
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/5d61bf57-956c-4cc7-bd0a-fe69c0dcf4ca)

IvanIKS commented:
I've changed "100" with variable "maxNumber" that represents the maximal number of ball. Can you please check it?

IvanIKS commented:
I'm sorry I did not commited it earlier.
